Fix grammar (#798)
hasura/graphql-engine: README.md

  - [Browse APIs](https://hasura-realtime-group-chat.herokuapp.com/)

> - Live location tracking app that shows a running vehicle changing current GPS

Reviewer: ```suggestion
- Live location tracking app that shows a running vehicle changing its current position
```

>   coordinates moving on the map. 

Reviewer: ```suggestion
  on a map. 
```

Hasura brand assets (logos, the Hasura mascot, powered by badges etc.) can be
found in the [assets/brand](assets/brand) folder. Feel free to use them in your

> application/website etc. We would be thrilled if you will add the "Powered by Hasura"

Reviewer: ```suggestion
application/website etc. We'd be thrilled if you added the "Powered by Hasura"
```
